app/common/forms/validators.py

```diff
@@ -0,0 +1,76 @@
+from wtforms.fields.core import Field
+from wtforms.form import BaseForm
+from wtforms.validators import ValidationError
+
+
+class MaxWords:
+    """
+    Validates that a field contains no more than a specified number of words.
+
+    Args:
+        max_words (int): Maximum number of words allowed
+        message (str, optional): Custom error message. If not provided, a default message is used.
+    """
```

Comment:
Don't love this comment either - the args section is fully redundant with the type hints. We should rely on the type hint information.